### DISCUSSION TITLE
Dashboard stuck in "Loading Statistics" mode after login

### DISCUSSION THREAD

**Discussion by JonEBee** [nextgenhealthcare/connect#5326] [Q&A]
Version 3.8.1 running on RHEL
Java version: 1.8.0_171

I logged into the engine and the dashboard has been "Loading Statistics" for the last 30 minutes.  Still missing all groups except the Default Group.

Is this a heap memory or DB issue?  This is my employer's primary production server.  There is a Load Balancer server in use, though I don't believe it is doing anything to affect the dashboard display.

Any suggestion or thoughts would be appreciated.

Thanks.

**Comment by pacmano1** [comment]
Close your session, modify it to show the java console (the radio button in the launcher).   

Also, what does mirth.log say?
